corehq/apps/app_manager/views/forms.py

```diff
@@ -576,6 +576,7 @@ def get_form_view_context_and_template(request, domain, form, langs, messages=me
             {'instanceId': instance.instance_id, 'instancePath': instance.instance_path}
             for instance in form.custom_instances
         ],
+        'can_preview_form': request.couch_user.has_permission(domain, 'edit_data')
```

Comment:
any reason not to combine `can_preview_form` and `allow_cloudcare`? i believe `allow_cloudcare` is only used in this template and they seem to have the same intent